### PR TITLE
Feature/sc 3560 text to var

### PR DESCRIPTION
**Pull request by abralurrahman**
- [x] Link zum Ticket https://ticketsystem.schul-cloud.org/browse/SC-3560

### CODE REVIEW

File: views/homework/submission.hbs [hunk 1]
```diff
@@ -90,9 +90,9 @@
 				{{#ifeq @root.currentUser._id @root.submission.studentId._id}}
 					<a href="/homework/submit/{{@root.submission._id}}/delete" class="btn btn-secondary btn-delete-submission" 
 						target="_blank"
-						data-method="delete-submission"
-						data-name="deine Abgabe"
-						alt="Abgabe löschen"
+						data-method='{{$t "homework.button.hrefDataMethod"  }}'
```
Comment (adrianjost) [inline views/homework/submission.hbs]:
```suggestion
						data-method="delete-submission"
```
this is not actual text content so it should be kept static

File: locales/de.json [hunk 2]
```diff
@@ -213,7 +213,10 @@
 		"button":{
 			"addTask": "Aufgabe hinzufügen",
 			"filterAdd": "Filter hinzufügen",
-			"filterApply": "Übernehmen"				        
+			"filterApply": "Übernehmen",
+			"hrefDataMethod": "delete-submission",
```
Comment (adrianjost) [inline locales/de.json]:
```suggestion
```

File: views/homework/submission.hbs [hunk 3]
```diff
@@ -90,7 +90,7 @@
 				{{#ifeq @root.currentUser._id @root.submission.studentId._id}}
 					<a href="/homework/submit/{{@root.submission._id}}/delete" class="btn btn-secondary btn-delete-submission" 
 						target="_blank"
-						data-method="delete-submission"
+						data-method= "delete-submission"
```
Comment (adrianjost) [inline views/homework/submission.hbs]:
```suggestion
						data-method="delete-submission"

```

File: views/homework/overview.hbs [hunk 4]
```diff
@@ -46,10 +46,10 @@
                     {{#content "body"}}
                         <div class="modal-footer">
                             <button type="button" class="btn btn-secondary btn-close">
-                                Abbrechen
+                                '{{$t "homework.button.abort"  }}'
```
Comment (adrianjost) [inline views/homework/overview.hbs]:
```suggestion
                                '{{$t "global.button.cancel"  }}'
```

File: views/homework/overview.hbs [hunk 5]
```diff
@@ -46,10 +46,10 @@
...
                             </button>
                             <button type="submit" class="btn btn-primary btn-submit">
-                                Löschen
+                               '{{$t "homework.button.clear"  }}'
```
Comment (adrianjost) [inline views/homework/overview.hbs]:
```suggestion
                               '{{$t "global.button.delete"  }}'
```

File: locales/de.json [hunk 6]
```diff
@@ -213,7 +213,15 @@
...
+			"filterApply": "Übernehmen",
+			"hrefDataName": "deine Abgabe",
+			"hrefAlt": "Abgabe löschen"	,
+			"abort"	: "Abbrechen",
+			"clear" : "Löschen"	        
+		} ,
+		"text":{
+             "noTasks" : "Keine Aufgaben"      
+
+		}
```
Comment (adrianjost) [inline locales/de.json]:
```suggestion
		"text":{
             "noTasks" : "Keine Aufgaben"      
		}
```

File: locales/de.json [hunk 7]
```diff
@@ -213,7 +213,15 @@
 		"button":{
 			"addTask": "Aufgabe hinzufügen",
 			"filterAdd": "Filter hinzufügen",
-			"filterApply": "Übernehmen"				        
-		} 
+			"filterApply": "Übernehmen",
+			"hrefDataName": "deine Abgabe",
+			"hrefAlt": "Abgabe löschen"	,
+			"abort"	: "Abbrechen",
+			"clear" : "Löschen"
```
Comment (adrianjost) [inline locales/de.json]:
```suggestion
```

File: views/homework/feedback.hbs [hunk 8]
```diff
@@ -1,20 +1,20 @@
 <section id="feedback" class="tab-content section-homeworkdetails">
     <div class="homework print">
         {{#if @root.submission.grade}}
-            Du hast <b class="grade">{{@root.submission.grade}}%</b> richtig gelöst.
+           '{{$t "homework.text.youHave" }}' <b class="grade">{{@root.submission.grade}}%</b> '{{$t "homework.text.solvedCorrectly" }}'
```
Comment (Tratori) [inline views/homework/feedback.hbs]: I Suggest leaving parts like these out for now and adding them to the code snippets on 
[our docs](https://docs.schul-cloud.org/pages/viewpage.action?pageId=113643606 )
as this way of dealing with variables will cause problems when translating to other languages.
We'll have to use some kind of interpolation here, i guess.

**Comment by malcm** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5209407&bid=17083758)

File: static/scripts/topicEdit.js [hunk 9]
```diff
@@ -934,7 +934,7 @@ class TopicNexboard extends TopicBlock {
                     content: content
                 });
                 return 0;
-        }});
+       }});
```
Comment (adrianjost) [inline static/scripts/topicEdit.js]:
```suggestion
        }});
```

**Pull request merged**